packages/translations/.eslintrc.js

```diff
@@ -2,5 +2,6 @@ module.exports = {
     extends: '../../.eslintrc.js',
     rules: {
         'import/no-unresolved': 'off',
+        'import/no-extraneous-dependencies': ['error', { devDependencies: true }],
```

Comment:
The default value of the `devDependencies` option is `true`. Is it needed to state that in the rule?

Reply:
Thank u for your revision Hamid. I tried that too but we need the default value of devDependencies inside this rule.